stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -907,6 +907,22 @@ end
 sqrt(A::AdjointAbsMat) = adjoint(sqrt(parent(A)))
 sqrt(A::TransposeAbsMat) = transpose(sqrt(parent(A)))
 
+# Cube root of a real-valued matrix
+function cbrt(A::AbstractMatrix{<:Real})
```

Comment:
Just add a docstring here, similar to `sqrt(A)`, explaining what the cube root of a matrix does.

Reply:
Will do!

Reply:
@stevengj Done! Thank you very much for your support. Please feel free to modify the docstring, or to suggest further improvements to the provided code.